docs/modules/sort/dataflow_example.md

```diff
@@ -7,305 +7,1717 @@ sidebar_position: 3
 
 To make it easier for you to create inlong-sort jobs, here we list some dataflow configuration examples.
 
-## Pulsar to Kafka
+## MySQL to Kafka
 
-Normal example:
+- Normal example:
```

Comment:
what `Normal` meaning is here? is it needless?

Reply:
It is needless,i remove it